tests/test_generic/tests/test_industry_requirements.py

```diff
@@ -77,7 +77,7 @@ def test_cloc_exclude_view(self):
     def test_sale_ok_and_is_published_in_db(self):
         models = ["product.template", "product.product"]
         for model in models:
-            if "is_published" in self.env[model]._fields:
+            if model in self.env and "is_published" in self.env[model]._fields:
```

Comment:
which module doesn't use product? :thinking:

Reply:
I do not remember but this raised an error.

Reply:
Really weird. Could you push again without this change to show the error?